cmd/stacker/check.go

```diff
@@ -39,6 +39,10 @@ func doCheck(ctx *cli.Context) error {
 
 	log.Infof("%s %s", config.RootFSDir, fstype)
 
+	if fstype == "NFS(6969)" {
+		return errors.Errorf("remote-mounted filesystem %s detected at %s", fstype, config.RootFSDir)
+	}
```

Comment:
I'd like to suggest an improvement to the error message:

    stacker roots dir (--roots-dir) path %s is not supported on NFS.

the string "NFS(6969)" has exaxtly zero relevant hits in my search, so there is no value in showing it to a user.  and specifying *which* config option it came from is important to fix the issue.

Reply:
```
$ ./stacker --roots-dir ~/tmp check
Linux 5.19.0-46-generic #47~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Jun 21 15:35:31 UTC 2
/home/rchincha/tmp NFS(6969)
error: roots dir (--roots-dir) path /home/rchincha/tmp is not supported on NFS.
error: exit status 1
```

Fixed.